src/WebsocketServer.hs

```diff
@@ -57,15 +60,16 @@ handleClient conn path core = do
   uuid <- newUUID
   let
     state = coreClients core
+    logRecords = coreLogRecords core
     onConnect = do
       modifyMVar_ state (pure . Subscription.subscribe path uuid conn)
-      putStrLn $ "Client " ++ (show uuid) ++ " connected, subscribed to " ++ (show path) ++ "."
+      log (T.pack $ "Client " ++ (show uuid) ++ " connected, subscribed to " ++ (show path) ++ ".") logRecords
```

Comment:
Is `T.pack` necessary? I'd expect the `OverloadedStrings` extension to take care of this.